tsconfig.json

```diff
@@ -7,7 +7,6 @@
         "packages/*/src"
       ]
     },
-    "target": "esnext",
```

Comment:
Now using base target across jellyfish, I suggest we override this in individual build configs if we need to.